gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general-ng.component.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Inject, OnInit } from '@angular/core';
+import { takeUntil, tap } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { FormControl, FormGroup } from '@angular/forms';
+
+import { ApplicationDetails } from './applicationDetails';
+
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+import { ApplicationGeneralNgService } from '../../../../../services-ngx/application-general-ng.service';
+import { UIRouterStateParams } from '../../../../../ajs-upgraded-providers';
+
+@Component({
+  selector: 'application-general-ng',
+  template: require('./application-general-ng.component.html'),
+  styles: [require('./application-general-ng.component.scss')],
+})
+export class ApplicationGeneralNgComponent implements OnInit {
+  public applicationDetails: ApplicationDetails;
+  public applicationForm: FormGroup;
```

Comment:
What do you think of keeping only this formGroup in public? and remove `applicationDetailsForm`, `applicationImagesForm`

you can access a child form group in html with `formGroupName`

Reply:
Done

Reply:
How do I access that in html template?
`<form *ngIf="applicationForm.details" [formGroup]="applicationForm" autocomplete="off" gioFormFocusInvalid>
  <mat-card class="details-card" [formGroup]="applicationForm.details">`
  This approach doesn't work

Reply:
Done